tests/integration/test_index_query/Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN apt-get update && \
 
 RUN python -m pip install --no-cache-dir --upgrade pip && \
     git clone https://github.com/jina-ai/jinad.git && \
-    pip install $(grep -ivE "jina" jinad/requirements.txt)
+    pip install $(grep -ivE "jina" jinad/requirements.txt) && \
+    cd jinad && python setup.py install
```

Comment:
Make sure that `jina` is not reinstalled again (so that version is not overriden from the one ìn base image) (maybe adding `--ignore-installed` can work